tmt/base.py

```diff
@@ -1714,13 +1721,44 @@ def import_plan(self) -> Optional['Plan']:
                 else:
                     tmt.utils.git_clone(plan_id.url, destination, self)
                 if plan_id.ref:
-                    self.run(Command('git', 'checkout', plan_id.ref), cwd=destination)
+                    # attempt to evaluate dynamic reference
+                    try:
+                        # Would it hurt if we simply rewrite plan_id.ref? Better choose safer path.
```

Comment:
@psss Well, that was basically the point of the comment, I don't know `tmt` code enough to say what would be the consequences. But definitely it can be simplified if you think it is safe to do.

Reply:
If I understand it correctly, here we apply the dynamic `ref` in a very similar way as we do in the `discover` step where it is overriden. I'd vote for simplifying the implementation and overwrite it here as well. @happz, do you have any concerns about this approach?

Reply:
No idea, TBH. I like the preservation of the original ref, therefore I for one don't see the extra attribute as a burden. I don't know what unexpected effect overwriting `ref` might have, if any; my knowledge of this area is still superficial at best :(